src/test/java/plugins/JobDslPluginTest.java

```diff
@@ -958,7 +958,7 @@ public void are_columns_created() {
         assertThat(view, containsColumnHeader("Last Success"));
         assertThat(view, containsColumnHeader("Last Failure"));
         assertThat(view, containsColumnHeader("Last Duration"));
-        assertThat(view, containsSvgWithText("Schedule a Build for " + job1.name));
+        assertThat(view, containsLinkWithTooltip("Schedule a Build for " + job1.name));
```

Comment:
doesn't appear to be forward compatible: https://ci.jenkins.io/blue/organizations/jenkins/Core%2Facceptance-test-harness/detail/PR-740/2/tests

Reply:
cc @janfaracik